batch-analysis/process-json.awk

```diff
@@ -0,0 +1,203 @@
+# This script reads in a json file and converts two particular JSON structs to inline
```

Comment:
This script is pretty neat but sadly a bit hard to maintain given the esoteric nature of awk. 

Can you explain a bit more about why this is needed? Is there an easier implementation we could do on top of existing tools (e.g. `jq`) ?

Reply:
fair point, I installed `jq` couldn't figure out how to do use it to do what I needed (i.e. a non-semantic formatting change that isn't totally minified or expanded)

Happy to redo this with a mainstream programming language if you want

Reply:
Also more curious on why this is needed at all -- is this purely for formatting or are there other reasons?

Reply:
It's just for formatting - makes it a lot easier to inspect output for large packages because all the identifiers and literals take up 1 line instead of 4

Reply:
In that case a complicated DFA implemented in awk seems like overkill :) This is just to aid in human readability as you read through the results for all the packages? 

If the intention of this is to enable easier human readability of the results, I'd suggest some lightweight wrapper in a more mainstream language to format things in the way you want. Or alternatively, if this is specific to you you can also just keep this uncommitted and use it locally :)

Reply:
I replaced the `awk` script with a Python script that is actually a modification of the `json.tool` component that was already used. It should be more maintainable.